Makefile

```
@@ -48,19 +48,23 @@ __start__: decrypt.${ENV}
 #     make decrypt.{dev,staging,prod} # Decrypt secrets for given environment
 #     make encrypt.{dev,staging,prod} # Encrypt secrets for given environment
 
-%: %.enc
-	@gcloud kms decrypt --ciphertext-file=$< --plaintext-file=$@ \
+.%.env .%.json %.pem .%.key: FORCE
+	@gcloud kms decrypt --ciphertext-file=cloudbuild/$@.enc --plaintext-file=$@ \
```

Why not:

```suggestion
%: cloudbuild/%.enc
	@gcloud kms decrypt --ciphertext-file=$< --plaintext-file=$@ \
```

That would make more sense to me.

---

yes just gave this a spin and it works great 👍 . I had put in the `FORCE` target as the previous iteration was not triggering the encryption recipe when I changed the decrypted file, but this one works better with the change tracking.